.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: cpp
```

Comment:
Top level Travis CI file. https://travis-ci.org/github/dilawar/stimfit 
Also github actions can also be used for CI/CD.